overlord/assertstate/assertmgr.go

```diff
@@ -110,6 +113,14 @@ func userFromUserID(st *state.State, userID int) (*auth.UserState, error) {
 	return auth.User(st, userID)
 }
 
+type assertionNotFoundError struct {
+	ref *asserts.Ref
+}
+
+func (e *assertionNotFoundError) Error() string {
+	return fmt.Sprintf("%s %v not found", e.ref.Type.Name, e.ref.PrimaryKey)
```

Comment:
@mvo5 I have been actually thinking of giving a String method to Ref, just couldn't commit to how it should look like